awx/ui_next/src/components/CredentialChip/CredentialChip.jsx

```diff
@@ -19,7 +19,9 @@ function CredentialChip({ credential, i18n, i18nHash, ...props }) {
   return (
     <Chip {...props}>
       <strong>{type}: </strong>
-      {credential.name}
+      {credential.kind === 'vault'
+        ? `${credential.name} | ${credential.inputs?.vault_id}`
+        : credential.name}
```

Comment:
@AlexSCorey It's possible for a vault credential to have no vault_id, in which case we'd want to hide the `|`